src/python_testing/TC_ICDM_2_1.py

```diff
@@ -15,98 +15,263 @@
 #    limitations under the License.
 #
 import logging
+from enum import IntEnum, IntFlag, auto
 
 import chip.clusters as Clusters
-from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main
+from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
 from mobly import asserts
 
-logger = logging.getLogger('PythonMatterControllerTEST')
-logger.setLevel(logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class UserActiveModeTriggerHintBitmap(IntFlag):
```

Comment:
Does this not get generated for you?

Reply:
It is but only learned about it when running the unit tests. Going to push an update next week with both updates.